src/components/MainPageMovieContainer.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import { Card, Container, Image, Grid } from 'semantic-ui-react';
+import axios from 'axios';
+import _ from 'lodash';
+import he from 'he';
+
+const MainPageMovieContainer = () => {
+	const [topTenMovies, setTopTenMovies] = useState([]);
+	const [errorMessage, setErrorMessage] = useState();
+	const fetchMovieData = async () => {
+		try {
+			const response = await axios.get('/movies');
+			setTopTenMovies(response.data.results);
+		} catch (error) {
+			setErrorMessage(error.message);
+		}
+	};
+
+	useEffect(() => {
+		fetchMovieData();
+	}, []);
+
+	return (
+		<Container>
+			<Grid data-cy="movie-container">
+				<Grid.Row columns={5} stretched padded>
+					{topTenMovies.map((movie, i) => (
+						<Grid.Column>
+							<Card data-cy={`movie-${i}`}>
+								<Image src={movie.img} />
+								<Card.Content>
+									<Card.Header data-cy="title-header">
+										{he.decode(movie.title)}
+									</Card.Header>
+									<Card.Description>
+										Rating: {_.round(movie.avgrating, 1)}
```

Comment:
Wouldn't `.toFixed(1)` give you the same output? Is it worth importing a library for this functionality? :)

Reply:
Yeah we didn't think about .toFixed(1) at the writing moment. But for now it can be good practice to use _. ;)
We'll keep that in our bags for the future.